### ISSUE TITLE
Does this run on localhost?

### ISSUE THREAD

**Issue by feugen24** [tpeczek/Demo.AspNetCore.PushNotifications#6]
Hi,

I debugged the code and basically it gets to 
await _pushClient.RequestPushMessageDeliveryAsync(subscription, message, cancellationToken);

with what seems as correct data it doesn't throw any exception but I don't get any message back on the client.

Thanks.

**Comment by tpeczek** [comment]
Yes this works on localhost (to be honest I've deployed it only once to make sure it works on remote host as well). Are you sure that everythign is alright with subscription, push notifications are not blocked in browser and that network communication is working alright (it does need to reach through network to push service)?

**Comment by feugen24** [comment]
Works both on firefox and on chrome, it was a local issue.

Not sure what happened but I was trying on chrome, it asked notification permission and I gave it then nothing, I checked notification settings and it was allowed, restarting chrome or disabling all plugins had no effect, but strangely it worked on firefox. So I asked a colleague and it worked for him in chrome so it appears that for some reason I didn't get notifications on chrome on any site although yesterday it worked...
Restarting windows solved the problem.

Thanks for the quick response( and code/blog :) ).
